db/migrate/20240508080834_rename_promotion_rules_to_conditions.rb

```diff
@@ -0,0 +1,13 @@
+class RenamePromotionRulesToConditions < ActiveRecord::Migration[7.1]
+  def change
+    rename_table :friendly_promotion_rules, :friendly_conditions
+    rename_table :friendly_promotion_rules_stores, :friendly_conditions_stores
+    rename_table :friendly_promotion_rules_taxons, :friendly_conditions_taxons
+    rename_table :friendly_promotion_rules_users, :friendly_conditions_users
+    rename_table :friendly_products_promotion_rules, :friendly_conditions_products
+    rename_column :friendly_conditions_stores, :promotion_rule_id, :condition_id
+    rename_column :friendly_conditions_taxons, :promotion_rule_id, :condition_id
+    rename_column :friendly_conditions_users, :promotion_rule_id, :condition_id
+    rename_column :friendly_conditions_products, :promotion_rule_id, :condition_id
+  end
```

Comment:
do you need to rename the type column values as well?

Reply:
yes